projects/angular/src/accordion/_variables.accordion.scss

```diff
@@ -4,34 +4,40 @@
-$clr-accordion-text-color-active: variables.$clr-global-font-color !default;
-$clr-accordion-text-color-selected: variables.$clr-global-font-color !default;
-$clr-accordion-panel-disabled-color: variables.$clr-color-neutral-500 !default;
-$clr-accordion-active-background-color: variables.$clr-global-selection-color !default;
-$clr-accordion-content-background-color: variables.$clr-color-neutral-0 !default;
-$clr-accordion-header-background-color: variables.$clr-color-neutral-50 !default;
-$clr-accordion-header-disabled-background-color: variables.$clr-color-neutral-200 !default;
-$clr-accordion-header-hover-background-color: variables.$clr-color-neutral-200 !default;
-$clr-accordion-header-active-background-color: variables.$clr-color-neutral-200 !default;
-$clr-accordion-border-left-color: variables.$clr-color-neutral-200 !default;
-$clr-accordion-border-left-current-color: $clr-accordion-border-left-color !default;
+$clr-accordion-text-color: var(--clr-accordion-text-color) !default;
+$clr-accordion-text-color-hover: var(--clr-accordion-text-color-hover) !default;
+$clr-accordion-text-color-active: var(--clr-accordion-text-color-active) !default;
+$clr-accordion-panel-disabled-color: var(--clr-accordion-panel-disabled-color) !default;
+$clr-accordion-active-background-color: var(--clr-accordion-active-background-color) !default;
+$clr-accordion-content-background-color: var(--clr-accordion-content-background-color) !default;
+$clr-accordion-header-background-color: var(--clr-accordion-header-background-color) !default;
+$clr-accordion-header-disabled-background-color: var(--clr-accordion-header-disabled-background-color) !default;
+$clr-accordion-header-hover-background-color: var(--clr-accordion-header-hover-background-color) !default;
+$clr-accordion-header-active-background-color: var(--clr-accordion-header-active-background-color) !default;
+
+$clr-accordion-header-button-padding: var(--clr-accordion-header-button-padding) !default;
+$clr-accordion-error-color: var(--clr-accordion-error-color) !default;
+$clr-accordion-complete-color: var(--clr-accordion-complete-color) !default;
+$clr-accordion-header-left-indicator: var(--clr-accordion-header-left-indicator) !default;
+$clr-accordion-header-left-current-indicator: var(--clr-accordion-header-left-current-indicator) !default;
+$clr-accordion-header-gap: var(--clr-accordion-header-gap) !default;
+$clr-accordion-content-color: var(--clr-accordion-content-color) !default;
+$clr-accordion-content-font-size: var(--clr-accordion-content-font-size) !default;
+$clr-accordion-content-padding: tokens.$cds-global-space-5 tokens.$cds-global-space-6 tokens.$cds-global-space-5
+  tokens.$cds-global-space-11 !default;
+
 $clr-accordion-responsive-breakpoint: map.get(variables.$clr-grid-breakpoints, sm) !default;
 $clr-accordion-error-color: variables.$clr-global-error-color !default;
 $clr-accordion-complete-color: variables.$clr-global-success-color !default;
```

Comment:
Use respective custom properties and core tokens.

Reply:
Duplicated now

Reply:
removed